.github/workflows/check.yml

```diff
@@ -0,0 +1,116 @@
+name: Check
+
+on:
+  pull_request:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9"]
+        django-version: ["2.2", "3.0", "3.1", "3.2", "4.0"]
+        exclude:
+          # Python 3.6 is not supported beyond Django 3.2
+          - python-version: "3.6"
+            django-version: "4.0"
+          # Python 3.7 is not supported beyond Django 3.2
+          - python-version: "3.7"
+            django-version: "4.0"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: pip
+          cache-dependency-path: poetry.lock
+
+      - name: Set up Poetry
+        uses: abatilo/actions-poetry@v2
+        with:
+          # Need to use version < 1.2.0 in order to support Python 3.6
```

Comment:
Ugh 😢 But 3.6 has been EOL for over a year.
Obviously, non-blocking complaint..

Reply:
Yeah, my goal here is to make this as close to what was in Circle as possible. At least in terms of behaviour. I'm following this up with a PR to update the supported versions (spoilers at https://github.com/thread/django-lightweight-queue/pull/87).